include/seqan3/search/fm_index/fm_index_cursor.hpp

```diff
@@ -423,6 +423,42 @@ class fm_index_cursor
         return index->index.comp2char[node.last_char] - 1; // text is not allowed to contain ranks of 0
     }
 
+    /*!\brief Returns the left bound in the suffix array.
+     * \returns Left suffix array bound.
+     *
+     * ### Complexity
+     *
+     * Constant.
+     *
+     * ### Exceptions
+     *
+     * No-throw guarantee.
+     */
+    size_type left_bound() const noexcept
```

Comment:
A general question: would `std::pair suffix_array_bounds` be a better function?

Reply:
Good idea, may be more concise. I'll ask Tobias.

Reply:
I'm not sure, we can probably leave it as is?
If we change it, I would rather like to return some struct `suffix_array_bounds` that has a `left` and `right` member.